src/UI/Settings/MediaManagement/Sorting/SortingViewTemplate.hbs

```diff
@@ -24,6 +24,30 @@
         </div>
     </div>
 
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Delete empty movie folders</label>
+
+        <div class="col-sm-9">
+            <div class="input-group">
+                <label class="checkbox toggle well">
+                    <input type="checkbox" name="deleteEmptyFolders"/>
+
+                    <p>
+                        <span>Yes</span>
+                        <span>No</span>
+                    </p>
+
+                    <div class="btn btn-primary slide-button"/>
+                </label>
+
+                <span class="help-inline-checkbox">
+                    <i class="icon-radarr-form-info" title="If this is enabled, folders that don't contain any movie files are deleted."/>
```

Comment:
I would make this a warning already. Also should probably be an advanced option.